repokid/__init__.py

```diff
@@ -11,15 +11,30 @@
 #     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #     See the License for the specific language governing permissions and
 #     limitations under the License.
-from collections import defaultdict
+import collections
+import datetime
 import inspect
 import json
 import logging.config
 import os
+import socket
+import sys
 
 import import_string
+import logmatic
+from pytz import timezone
 
-__version__ = "0.10.0"
+__version__ = "0.10.1"
```

Comment:
I think we want a minor version update for log format changing

Reply:
Oops, fixed thanks